cumulusci/tasks/salesforce/content_documents.py

```diff
@@ -0,0 +1,209 @@
+import base64
+from pathlib import Path
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg
+from cumulusci.core.exceptions import CumulusCIException
+from simple_salesforce.exceptions import SalesforceMalformedRequest
+
+
+def to_cumulusci_exception(e: SalesforceMalformedRequest) -> CumulusCIException:
+    return CumulusCIException(
+        "; ".join([error.get("message", "Unknown.") for error in e.content])
+    )
+
+
+class InsertContentDocument(BaseSalesforceApiTask):
+    task_docs = """
+Uploads a profile photo for a specified or default User.
+
+Examples
+--------
+
+Upload a profile photo for the default user.
+
+.. code-block:: yaml
+
+    tasks:
+        upload_profile_photo_default:
```

Comment:
This needs to be updated.  It was copied from `cumulusci.tasks.salesforce.user.photos`.